.github/workflows/check-nest-lint.yaml

```diff
@@ -0,0 +1,33 @@
+name: 'Check - Nest Lint'
+
+on:
+  workflow_call:
+    inputs:
+      node_version:
+        type: string
+        description: "Version to use for node"
+        required: true
+      deploy_stage:
+        type: string
+        description: "Deployment stage"
+        required: true
+env:
+  NODE_VERSION: ${{ inputs.node_version }}
+  DEPLOY_STAGE: ${{ inputs.deploy_stage }}
```

Comment:
Is the deploy_stage variable/env used anywhere?

Reply:
The variable is needed when running the tests, if not set an error is thrown.

Reply:
Tested here: https://github.com/dBildungsplattform/dbildungs-iam-server/actions/runs/6574557741/job/17859846102

Reply:
nice to know that everyone is wondering about the same thing :D